### PR TITLE
Upgrade to mongoid 9

### PR DESCRIPTION
**Pull request by SairamSrinivasan**
PR to upgrade monogid to v9 as it was just released: https://www.mongodb.com/docs/mongoid/master/release-notes/mongoid-9.0/

### CODE REVIEW

**Comment by SairamSrinivasan** [comment]
I ran rspec and all the tests pass.

**Review by dblock — CHANGES_REQUESTED** [review]
Glad it works out of the box! Add to CI, https://github.com/mongoid/mongoid-locker/blob/master/.github/workflows/test.yml#L28, and CHANGELOG please?

**Comment by SairamSrinivasan** [comment]
@dblock Thanks for the review. Can we release this as v3.0.0, bumping the version to a MAJOR?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ## Changelog
 
 ### 2.0.3 (Next)
-
+* [#101](https://github.com/mongoid/mongoid-locker/pull/101): Replace the $where operator with the $expr operator - [@amanfrinati](https://github.com/amanfrinati).
 * [#102](https://github.com/mongoid/mongoid-locker/pull/102): Re-added danger, removed Travis-CI config, extracted RuboCop - [@dblock](https://github.com/dblock).
 * [#104](https://github.com/mongoid/mongoid-locker/pull/104): Fixed code coverage - [@dblock](https://github.com/dblock).
-* [#101](https://github.com/mongoid/mongoid-locker/pull/101): Replace the $where operator with the $expr operator - [@amanfrinati](https://github.com/amanfrinati).
-* Your contribution here.
+* [#105](https://github.com/mongoid/mongoid-locker/pull/105): Upgraded supported mongoid version to version 9.0 - [@saisrinivasan](https://github.com/SairamSrinivasan)
```
Comment (dblock) [inline CHANGELOG.md:7]: Change to "Added support for Mongoid 9.0". Don't forget the period at the end or danger will complain.

**Review by dblock — CHANGES_REQUESTED** [review]
This is not a breaking change, so not 3.0, but let's bump version to 2.1.0 as part of this PR since it's a new feature.

**Comment by dblock** [comment]
Looks good, thank you. Will cut a release soon-ish unless someone beats me to it. Btw, lots of other mongoid- gems in this org could use similar help!

**Comment by dblock** [comment]
I released 2.1.0.

**Pull request merged**